doc/user/content/transform-data/idiomatic-materialize-sql/appendix/idiomatic-sql-chart.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Idiomatic Materialize SQL chart"
+description: "Cheatsheet of idiomatic Materialize SQL."
+menu:
+  main:
+    parent: idiomatic-materialize-appendix
+    weight: 10
+    identifier: idiomatic-materialize-appendix-idiomatic-materialize-sql-chart
+---
```

Comment:
This content could actually have been the landing page of the idiomatic-materialize-sql/ ... but, until I get a better understanding of the other content (in future work), figured a separate page will give me more flexibility in the future (not necessarily less work ... but keeping options open for now).